forge-gui/res/cardsfolder/upcoming/traumatic_prank.txt

```diff
@@ -0,0 +1,9 @@
+Name:Traumatic Prank
+ManaCost:2 R
+Types:Sorcery
+A:SP$ GainControl | ValidTgts$ Creature | TgtPrompt$ Select target creature | LoseControl$ EOT | AddKWs$ Haste | Untap$ True | SubAbility$ DBEffect | SpellDescription$ Gain control of target creature until end of turn. Untap that creature. It perpetually gains haste, "This creature can't block," and "At the beginning of your upkeep, this creature deals 1 damage to you."
```

Comment:
I'd remove `AddKWs$ Haste` here since the perpetual effect handles that

Reply:
While you're at it you can remove `TgtPrompt$ Select target creature` as Forge can parse that automatically